### PR TITLE
Convenience wrappers I call "ceph-cli"

### PR DESCRIPTION
**Pull request by hookenz**
I've created a few convenience wrappers which I hope you may find useful.
Currently I have ceph and ceph-disk.

They are simply shell scripts around docker run and can be installed in much the same way as systemd-docker.  Which is where I got the idea.

If you think this is a good way to use ceph from the host going forward, feel free to add some more, or I'll do so.

### CODE REVIEW

File: cli/ceph-disk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/bin/docker run --rm --privileged=true -v /etc/ceph:/etc/ceph -v /dev:/dev ceph/base ceph-disk "$@"
```
Comment (leseb) [inline cli/ceph-disk]: I think you should also expose `var/lib/ceph/bootstrap-osd` so `ceph-disk` can add new OSDs using the bootstrap key.
I did some (unsuccessful) experiment trying to get `ceph-disk activate` I'll be happy to hear if you got it working :).

Comment (hookenz) [inline cli/ceph-disk]: I'll take a look.

Comment (leseb) [inline cli/ceph-disk]: I just managed to create OSDs with `ceph-disk` somehow using `-v /dev:/dev` instead of `-v /dev/sdb:/dev/sdb worked.`

**Comment by hookenz** [comment]
Yeah. After that I'm temp mounting the disk and putting the osd is into the whoami file.  Then id you start the ceph osd it'll be initialised.  That process is what ceph-disk activate does. I'm thinking of emulating that process when you call activate   with ceph disk because the real activate starts the things differently than our server.  I've got an idea on how to do that

**Comment by hookenz** [comment]
Excuse the spelling and grammar. Wrote that in on my phone.  Hope it makes sense.

**Comment by Ulexus** [comment]
I like this; I use something similar (but simpler) on my workstation:

```
#!/bin/bash

ARGS=""
if [ $# -gt 0 ]; then
   ARGS="ceph $*"
else
   ARGS="bash"
fi
ssh -t $CEPH_MON docker exec -it ceph-mon $ARGS
```

I don't know that I would call it `cli`, though.  My hunch would be to create an `/examples/coreos/` directory and file it there as `tools`.

**Comment by hookenz** [comment]
Had thought it calling it tools later.  We could do that.

I also was thinking of making it start a container that would use docker exec.  That way it would work on any monitor or ceph node.  As some tools like ceph-disk you want on a node.  but if we remove the admin key then ceph status won't work on them.

**Comment by hookenz** [comment]
I've rearranged everything as suggested.  I've also added my mount-osds script which you may find useful.  It's how I'm mounting them under CoreOS.

**Comment by Ulexus** [comment]
LGTM; @leseb ?

**Comment by leseb** [comment]
Thanks @hookenz!

**Pull request merged**